libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -58,52 +48,40 @@ export const AUTOCOMPLETE_NAV_KEYS = new Set(['Enter', 'ArrowUp', 'ArrowDown']);
     },
   ],
 })
-export class AutocompleteComponent
-  implements OnInit, OnDestroy, OnChanges, ControlValueAccessor {
-  @Input() field: Option;
-
+export class AutocompleteComponent implements OnInit, ControlValueAccessor {
```

Comment:
I don't really think this needs to be a ControlValueAccessor with the input connected thru ControlContainer, but I'll save that for later. Same with FieldComponent; seems like it should just be a pass thru to child form components using the built in ControlValueAccessors.